<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSView</class>
 <widget class="QDockWidget" name="SSView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentation Slice View</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbDir">
         <item>
          <property name="text">
           <string>XY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>XZ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sliceLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Slice#</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbNum">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rotationLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Rot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sbRot">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pushMov">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../open_iA/core/open_iA.qrc">
           <normaloff>:/images/savemov.png</normaloff>:/images/savemov.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pushSave">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../open_iA/core/open_iA.qrc">
           <normaloff>:/images/savescreen.png</normaloff>:/images/savescreen.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbSettings">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../open_iA/core/open_iA.qrc">
           <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="slicerWidget" native="true"/>
      </item>
      <item>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../open_iA/core/open_iA.qrc"/>
 </resources>
 <connections/>
</ui>
